DataFormats/BTauReco/interface/TemplatedSoftLeptonTagInfo.h

```diff
@@ -25,7 +25,8 @@ class SoftLeptonProperties {
         deltaR(   std::numeric_limits<float>::quiet_NaN() ),
         ratio(    std::numeric_limits<float>::quiet_NaN() ),
         ratioRel( std::numeric_limits<float>::quiet_NaN() ),
-        elec_mva( std::numeric_limits<float>::quiet_NaN() )
+        elec_mva( std::numeric_limits<float>::quiet_NaN() ),
+        charge(   std::numeric_limits<int>::quiet_NaN() )
```

Comment:
quiet_NaN for int is 0.
I suppose charge=0 never becomes a valid result.
Please confirm.